.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,4 +12,17 @@
 
 **PR Content:**
 
-* 
+**What type of change are you bringing in this PR?**
+- [ ] Backend changes
+- [ ] UI changes
+- [ ] Other
+
+**If possible, what kind of change it is (in detail)?**
+<!-- Remove this comment and answer here -->
+
+**Did you carefully checked your PR for bugs?**
+- [ ] Yes
+- [ ] No
```

Comment:
* Change **checked** to **check**

* Add another option - **Not applicable**

Reply:
@alphaX86 refer point 1 ^^^

Reply:
One sec... Wait here

Reply:
ERROR removed

Reply:
@lordlabuckdas you can merge now